docs/pages/server-access/guides/openssh.mdx

```diff
@@ -4,394 +4,420 @@ description: How to use Teleport on legacy systems with OpenSSH and sshd.
 ```
 
-Restart `sshd`.
+Copy the output. 
 
-Now, `sshd` will trust users who present a Teleport-issued certificate.
-The next step is to configure host authentication.
+On the host where you are running `sshd`, run the following commands.
 
-The recommended solution is to ask Teleport to issue valid host certificates for
-all OpenSSH nodes. To generate a host certificate, run the following `tctl` command:
+Assign the output of the `tctl auth export` command to an environment variable:
 
 ```code
-# Creating host certs, with an array of every host to be accessed.
-# Wildcard certs aren't supported by OpenSSH. The domain must be fully 
-# qualified.
-# Management of the host certificates can become complex. This is another
-# reason we recommend using Teleport SSH on nodes.
-$ sudo tctl auth sign \
-      --host=api.example.com,ssh.example.com,64.225.88.175,64.225.88.178 \
-      --format=openssh \
-      --out=api.example.com
-
-The credentials have been written to api.example.com, api.example.com-cert.pub
-
-# You can use ssh-keygen to verify the contents.
-$ ssh-keygen -L -f api.example.com-cert.pub
-#api.example.com-cert.pub:
-#        Type: ssh-rsa-cert-v01@openssh.com host certificate
-#        Public key: RSA-CERT SHA256:ireEc5HWFjhYPUhmztaFud7EgsopO8l+GpxNMd3wMSk
-#        Signing CA: RSA SHA256:/6HSHsoU5u+r85M26Ut+M9gl+HventwSwrbTvP/cmvo
-#        Key ID: ""
-#        Serial: 0
-#        Valid: after 2020-07-29T20:26:24
-#        Principals:
-#               api.example.com
-#               ssh.example.com
-#               64.225.88.175
-#               64.225.88.178
-#        Critical Options: (none)
-#        Extensions:
-#                x-teleport-authority UNKNOWN OPTION (len 47)
-#                x-teleport-role UNKNOWN OPTION (len 8)
+$ export KEY="<pasted output>"
 ```
 
-Then add the following lines to `/etc/ssh/sshd_config` on all OpenSSH nodes, and
-restart `sshd`.
+Make the public key accessible to `sshd`:
 
-```yaml
-HostKey /etc/ssh/api.example.com
-HostCertificate /etc/ssh/api.example.com-cert.pub
+```code
+$ sudo bash -c "echo \"$KEY\" > /etc/ssh/teleport_user_ca.pub"
+$ sudo bash -c "echo 'TrustedUserCAKeys /etc/ssh/teleport_user_ca.pub' >> /etc/ssh/sshd_config"
 ```
 
-## Use the OpenSSH client to access Teleport Nodes
+Restart `sshd`. For systemd-enabled hosts, run the following command:
+
+```code
+$ sudo systemctl restart sshd
+```
 
-It is possible to use the OpenSSH client `ssh` to connect to Nodes within a
-Teleport cluster. Teleport supports SSH subsystems and includes a `proxy` subsystem that can be used like `netcat` is with `ProxyCommand` to connect
-through a jump host.
+Now, `sshd` will trust users who present a Teleport-issued certificate.
 
-OpenSSH client configuration may be generated automatically by `tsh`, or it can
-be configured manually. In either case, make sure you are running OpenSSH's
-`ssh-agent`, and have logged in to the Teleport Proxy Service:
+## Step 2/4. Configure host authentication
```

Comment:
I think it would be nice if we provided a short sentence that explained *why* you need to create a host certificate for the `sshd` host. Someone with less knowledge of `sshd` might be confused as to why the previous step is not enough, and I think it's nice for users to come away with a better understanding of Linux security fundamentals after reading our guides.

Reply:
I've added more context to the guide